app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+
+        visitor_date = visitor["vaccine"]["expiration_date"]
+        today = date.today()
+        if visitor_date < today:
+            raise OutdatedVaccineError
+
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError
```

Comment:
This is a Code Style issue. You are raising `NotWearingMaskError` without any message. It is a good practice to provide descriptive error messages for exceptions. You might want to raise it like this: `raise NotWearingMaskError('The visitor is not wearing a mask.')`